TESTS/Ace Deal Manager 2.1.1 - Default AFR - Workflow and updated test cases/FT Ace Deal Manager 2.1.1 - Default AFR - Workflow and updated test cases/ADM - Workflow Scenario 3  Edit and delete an existing deal - Critical/4/Input.md

```diff
@@ -0,0 +1,15 @@
+_Editing addition details:-_
+
+_Repeat 1.1 - 1.3 steps and do not save the deal _
+
+  
+1\. Edit details ( Expected Close date and Project Alias ) in details sections.  
+  
+2. Remove some old and add a new team member, participants ( accounts ) and deal contacts in people section
+
+  
+3\. Add new products and enter the value in expected fees of product revenue section of each product and verify that that summation value of the field of all products is updated in 'Total Expected Fees' in details sections.  
```

Comment:
The input or expected result should be updated, as the 'Total Expected Fees' does not update live - it requires user to first save (this is unclear)